improver/feels_like_temperature.py

```diff
@@ -133,74 +120,36 @@ def calculate_apparent_temperature(temperature, wind_speed,
     the WetBulbTemperature plugin which makes use of the Goff-Gratch method.
 
     Args:
-      temperature (iris.cube.Cube):
-        Cube of air temperatures
-
-      wind_speed (iris.cube.Cube):
-        Cube of 10m wind speeds
-
-      relative_humidity (iris.cube.Cube):
-        Cube of relative humidities
-
-      pressure (iris.cube.Cube):
-        Cube of air pressure
+        temperature (numpy.ndarray):
```

Comment:
Is this method used elsewhere, or should it be private like the _feels_like_temperature method?

Reply:
Both the apparent temperature and wind chill functions could be private (in fact my thoughts have been that this would be better as a class, but I wasn't intending to make quite such sweeping changes when I started this work).  I'll make them private.